infra/assembler.go

```diff
@@ -36,12 +37,15 @@ func (a *Assembler) sign(e *Elecments) *Elecments {
 	return e
 }
 
-func (a *Assembler) StartSigner(raw, signed chan *Elecments, done <-chan struct{}) {
+func (a *Assembler) StartSigner(raw chan *Elecments, signed []chan *Elecments, done <-chan struct{}) {
```

Comment:
I had a typo in `Elecments`... but let's fix in another PR

Reply:
Renaming in other pr, skip for now.